cloudify-manager-worker/README.md.gotmpl

```diff
@@ -0,0 +1,636 @@
+[//]: # -*- markdown -*-
```

Comment:
Is this needed?

Reply:
It is not necessary, just a comment for my text editor to work with this "README.md.gotmpl" file as with markdown file.
Actually I thought it shouldn't be rendered in HTML, but I was wrong, I'll change this behaviour.

Reply:
Fixed, for now this comment isn't rendered in HTML.